pkg/operation/botanist/component/vpnshoot/vpnshoot.go

```diff
@@ -684,6 +708,41 @@ func (v *vpnShoot) getVolumes(secretCA, secret, secretTLSAuth, secretDH *corev1.
 				},
 			},
 		})
+	} else {
+		volumes = append(volumes, corev1.Volume{
+			Name: volumeNameDevNetTun,
+			VolumeSource: corev1.VolumeSource{
+				HostPath: &corev1.HostPathVolumeSource{
+					Path: volumeMountPathDevNetTun,
+					Type: (*corev1.HostPathType)(pointer.String(string(corev1.HostPathCharDev))),
+				},
+			},
+		})
 	}
 	return volumes
 }
+
+func (v *vpnShoot) getInitContainers() []corev1.Container {
+	if !v.values.ReversedVPN.Enabled {
+		return []corev1.Container{}
+	}
+	return []corev1.Container{
+		{
+			Name:            initContainerName,
+			Image:           v.values.Image,
+			ImagePullPolicy: corev1.PullIfNotPresent,
+			Env:             v.getEnvVars(true),
+			SecurityContext: &corev1.SecurityContext{
+				Privileged: pointer.Bool(true),
+			},
+			Resources: corev1.ResourceRequirements{
+				Requests: corev1.ResourceList{
+					corev1.ResourceCPU:    resource.MustParse("100m"),
+					corev1.ResourceMemory: resource.MustParse("100Mi"),
+				},
+				Limits: v.getResourceLimits(),
+			},
+			VolumeMounts: v.getVolumeMounts(true),
```

Comment:
Why the init container needs to mount any volumes?
Isn't it simply doing
```
echo 7200 > /proc/sys/net/ipv4/tcp_keepalive_time
echo 75 > /proc/sys/net/ipv4/tcp_keepalive_intvl
echo 9 > /proc/sys/net/ipv4/tcp_keepalive_probes
echo 1 > /proc/sys/net/ipv4/ip_forward
```

?

```suggestion
```

Reply:
Adapted as requested with latest commit.